.github/workflows/build_and_push.yml

```diff
@@ -0,0 +1,79 @@
+
+name: Build and Push Container to ECR, deploy to lambda
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master]
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  REGISTRY: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/notify/api
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      images: ${{ steps.filter.outputs.changes }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            api: 'app/**'
```

Comment:
This action should run when any of the api code changes. I changed this from `api: 'api/**'` since our api code is in the app folder. Does the key name here (`api`) get used for anything?